### PR TITLE
Notification about new Risk Status shows when App is open (EXPOSUREAPP-10764)

### PR DESCRIPTION
**Pull request by mtwalli**
## Testing
- Use test menu to change risk level
- Notice that notification is triggered now in foreground
- To test the badge, it is a little bit tricky ,since you will land in home screen anyway after test menu

## Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10764

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/main/CWASettings.kt [hunk 1]
```diff
@@ -91,8 +91,6 @@ class CWASettings @Inject constructor(
         private const val PKEY_DEVICE_TIME_FIRST_RELIABLE = "devicetime.correct.first"
         private const val PKEY_DEVICE_TIME_LAST_STATE_CHANGE_TIME = "devicetime.laststatechange.timestamp"
         private const val PKEY_DEVICE_TIME_LAST_STATE_CHANGE_STATE = "devicetime.laststatechange.state"
-        private const val PKEY_NOTIFICATIONS_RISK_ENABLED = "notifications.risk.enabled"
-        private const val PKEY_NOTIFICATIONS_TEST_ENABLED = "notifications.test.enabled"
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/main/CWASettings.kt:95]: not used

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/TracingSettings.kt [hunk 2]
```diff
@@ -41,29 +41,20 @@ class TracingSettings @Inject constructor(@AppContext private val context: Conte
...
-//            revokeConsentToSubmission()
-//            submissionSettings.registrationToken.update { null }
-//            submissionSettings.devicePairingSuccessfulAt = null
-//            tracingSettings.initialPollingForTestResultTimeStamp = 0L
-//            submissionSettings.initialTestResultReceivedAt = null
-//            submissionSettings.isAllowedToSubmitKeys = false
-//            tracingSettings.isTestResultAvailableNotificationSent = false
-//            submissionSettings.isSubmissionSuccessful = false
-//            testResultDataCollector.clear()
-//        }
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/TracingSettings.kt:58]: clean up

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainActivity.kt [hunk 3]
```diff
@@ -107,8 +107,11 @@ class MainActivity : AppCompatActivity(), HasAndroidInjector {
             fabTooltip.close.setOnClickListener { viewModel.dismissTooltip() }
 
             scannerFab.apply {
-//                setShowMotionSpecResource(R.animator.fab_show)
-//                setHideMotionSpecResource(R.animator.fab_hide)
+                // TODO: Enable transition, Transition to scanner is disabled,
+                //  because it is causing a native crash in Camera PreviewView [CameraX]
+                //  see https://github.com/corona-warn-app/cwa-app-android/pull/4648#issuecomment-1005697916
+                //  setShowMotionSpecResource(R.animator.fab_show)
+                //  setHideMotionSpecResource(R.animator.fab_hide)
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainActivity.kt:114]: missing todo

**Review by AlexanderAlferov — DISMISSED** [review]
✅ Notification in foreground
✅ Notification badge over "Status" icon

**Review by chiljamgossow — APPROVED** [review]
LGTM

**Review by AlexanderAlferov — APPROVED** [review]

**Pull request merged**